src/libraries/AccountAccessParser.sol

```diff
@@ -271,10 +280,7 @@ library AccountAccessParser {
         // Get all storage writes as a state diff.
         address[] memory uniqueAddresses = getUniqueWrites({accesses: _accountAccesses, _sort: false});
 
```

Comment:
NIT: we maybe want to have comment on top here like previously to explain this array?

Reply:
yea will add in